plugins/chat/app/controllers/api/chat_channels_controller.rb

```diff
@@ -61,7 +61,7 @@ def update
 
   def find_chat_channel
     chat_channel = ChatChannel.find(params.require(:chat_channel_id))
-    guardian.ensure_can_see_chat_channel!(chat_channel)
+    guardian.ensure_can_join_chat_channel!(chat_channel)
```

Comment:
no tests need to change for this?

Reply:
Because guardian is tested, I added a test which ensures that guardian is invoked